platform/config/config-mapping-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+    <reference id="config"
+               interface="org.codice.ddf.config.ConfigService"
+               availability="mandatory"/>
+
+    <reference-list id="mapping.listeners"
+                    interface="org.codice.ddf.config.mapping.ConfigMappingListener"
+                    availability="optional"/>
```

Comment:
:question: Is this really optional?

Reply:
Why would I need a listener to be present for the mapping service to start? At the end of the day, the mapping service doesn't care who is using its service. Getting access to the mapping service is required for the OSGI admin agent.